app/presenters/check_answers/base.rb

```diff
@@ -59,5 +59,14 @@ def process_boolean_value(boolean_field:, value_field:, boolean_key:, value_key:
           }
         end)].compact
     end
+
+    def format_total(value)
+      text = "<strong>#{currency_value(value)}</strong>"
```

Comment:
I'm not qualified to review this PR, but since this class already includes TagHelper, which can make sanitised HTML tags for you, it's arguably more Rails-y to make this a 1-liner:

```ruby
def format_total(value)
  tag.strong(currency_value(value))
end
```

Reply:
Thanks for the pointer! More qualified than myself :D Have updated